sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -261,6 +261,17 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val HADOOPFSRELATION_SIZE_FACTOR = buildConf(
+    "org.apache.spark.sql.execution.datasources.sizeFactor")
```

Comment:
Is this config for all data sources or only hadoopFS-related data sources?

Reply:
this is only for HadoopFSRelation